python_modules/dagster/dagster/_cli/api.py

```diff
@@ -98,7 +100,7 @@ def _metrics_polling_interval(
     try:
         return float(
             dagster_run.tags.get(
-                "dagster/run_metrics_polling_interval_seconds",
+                ".dagster/run_metrics_polling_interval_seconds",
```

Comment:
... that are still undocumented and weren't communicated to users. good opportunity to move them to hidden tags as well.